roles/openshift_logging/tasks/generate_configmaps.yaml

```diff
@@ -1,10 +1,20 @@
 ---
 - block:
-    - copy:
-        src: elasticsearch-logging.yml
+    - fail:
```

Comment:
Moved to `install_logging`.  The validation already occurs in the when statement with taking the difference of the list against the acceptable values as defined in `vars/main.yaml`